pygeoapi/provider/postgresql.py

```diff
@@ -265,6 +265,44 @@ def query(self, startindex=0, limit=10, resulttype='results',
 
             return feature_collection
 
+    def get_previous(self, cursor, identifier):
+        """
+        Query previous ID given current ID
+
+        :param identifier: feature id
+
+        :returns: feature id
+        """
+        sql = 'SELECT {} AS id FROM {} WHERE {}<%s ORDER BY {} DESC LIMIT 1'
+        cursor.execute(SQL(sql).format(
+            Identifier(self.id_field),
+            Identifier(self.table),
+            Identifier(self.id_field),
+            Identifier(self.id_field),
+        ), (identifier,))
+        item = cursor.fetchall()
+        id = item[0]['id']
```

Comment:
`id` is a Python built-in function.  Suggest using `id_`.

Reply:
Would you like me to change that? I think it's more guaranteed to work the way it is. Also, it might be easier to read for someone like me who didn't know this. So, it is just an estaetic improvement, no?

Reply:
Given `id` is a Python [function](https://docs.python.org/3/library/functions.html#id) it would be safer not to override function names in the case that they be used elsewhere in the codebase.

Reply:
Agree to this @tomkralidis.